Makefile

```diff
@@ -45,19 +44,6 @@ sdk:
 		@ocaml/sdk-gen/csharp/generate \
 		@ocaml/sdk-gen/java/generate \
 		@ocaml/sdk-gen/powershell/generate
-	mkdir -p $(XAPISDK)/c
```

Comment:
Why have these lines been removed?

Reply:
I think we should keep these lines in the Makefile (at least for now). Is there another way to factor out the `dune build` lines?

Reply:
I've reintroduced the lines

Reply:
What exactly is the issue with these lines? Is it just repetitiveness (the logic is the same as the doc target)?